eng/deploy.yaml

```
@@ -230,14 +225,25 @@ stages:
       inputs:
         useGlobalJson: true
 
+    - task: AzureCLI@2
+      inputs:
+        azureSubscription: 'secret-manager-scenario-tests'
+        addSpnToEnvironment: true
+        scriptType: 'ps'
+        scriptLocation: 'inlineScript'
+        inlineScript: |
+          echo "##vso[task.setvariable variable=ARM_CLIENT_ID]$env:servicePrincipalId" 
+          echo "##vso[task.setvariable variable=ARM_TENANT_ID]$env:tenantId"
+          echo "##vso[task.setvariable variable=ARM_ID_TOKEN]$env:idToken"
```

```suggestion
          echo "##vso[task.setvariable variable=ARM_ID_TOKEN;iSsecret=true]$env:idToken"
```
Even though AzDO is doing the right thing right now, I'd like to see `isSecret=true` added to at least this line so that our intentions are clear.